### PR TITLE
Updates for lightning receive

### PR DESCRIPTION
**Pull request by thisconnect**
Note to myself:
- copy button is missing

### CODE REVIEW

File: frontends/web/src/routes/account/send/send.tsx [hunk 1]
```diff
@@ -652,9 +652,7 @@ class Send extends Component<Props, State> {
                     <Input
                       label={t('note.title')}
                       labelSection={
-                        <span className={style.labelDescription}>
-                          {t('note.input.description')}
-                        </span>
+                        <OptionalLabel>{t('note.input.description')}</OptionalLabel>
```
Comment (dangeross) [inline frontends/web/src/routes/account/send/send.tsx:655]: Should we change this to `lightning.send.description.optional`, or move this to `lightning.description.optional` ?

File: frontends/web/src/routes/lightning/receive/receive.tsx [hunk 2]
```diff
@@ -121,87 +122,116 @@ export function Receive({ accounts, code }: Props) {
...
-          </Column>
-          <Column>
-            <ColumnButtons className="m-top-default m-bottom-xlarge" inline>
-              <Button secondary onClick={back} disabled={busy}>
-                {t('button.back')}
-              </Button>
-            </ColumnButtons>
-          </Column>
-        </Grid>
+        <Spinner text="Hardcode invoice creating going…" guideExists={false} />
```
Comment (dangeross) [inline frontends/web/src/routes/lightning/receive/receive.tsx]: Is this text shown? If so we should add a translation key

File: frontends/web/src/routes/lightning/receive/receive.tsx [hunk 3]
```diff
@@ -121,87 +122,116 @@ export function Receive({ accounts, code }: Props) {
...
+          <ViewContent textAlign="center">
+            <Grid col="1">
+              <Column>
+                <h1 className={styles.title}>{t('lightning.receive.invoice.title')}</h1>
+                <div>
+                  <QRCode data={receivePaymentResponse?.lnInvoice.bolt11} />
+                </div>
+                <div className={styles.invoiceSummary}>
+                  {amountSatsText} sats
+                  (--- EUR)
```
Comment (dangeross) [inline frontends/web/src/routes/lightning/receive/receive.tsx:206]: I imagine this is placeholder for the fiat conversion. FYI I have been tasked to include the conversion rates in the views also later.

File: frontends/web/src/routes/account/send/send.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/routes/account/send/send.tsx:655]: This is not the lightning send but the onchain send component, that I refactored so we can use the same component+styles here and in lightning receive.

File: frontends/web/src/routes/lightning/receive/receive.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/routes/lightning/receive/receive.tsx]: Yes definitely, I changed to "Creating invoice"  and fixed it into the last commit.

File: frontends/web/src/routes/account/send/send.tsx (hunk 1 above)
Comment (dangeross) [reply frontends/web/src/routes/account/send/send.tsx:655]: Hah, ok sorry. Didn't check the filename

File: frontends/web/src/routes/lightning/receive/receive.tsx (hunk 2 above)
Comment (dangeross) [reply frontends/web/src/routes/lightning/receive/receive.tsx]: Sounds good

File: frontends/web/src/routes/lightning/receive/receive.tsx (hunk 3 above)
Comment (thisconnect) [reply frontends/web/src/routes/lightning/receive/receive.tsx:206]: lets do fiat conversion after PR to upstream/staging-ln

**Comment by thisconnect** [comment]
One more comment, in send PR I simplified some of the useEffect's and called parseInput directly. I think here we could also simplified it a bit. I didn't want to break anything but will look into reducing some of the state/effects soonish. (But just FYI, this should not block anything)

**Comment by thisconnect** [comment]
@dangeross I think I addressed all of your comments PTAL

**Review by dangeross — APPROVED** [review]
LGTM

**Pull request merged**